presto-spi/src/test/java/io/prestosql/spi/block/TestLongArrayBlockEncoding.java

```diff
@@ -13,48 +13,35 @@
  */
 package io.prestosql.spi.block;
 
-import io.airlift.slice.DynamicSliceOutput;
+import io.prestosql.spi.type.Type;
 import org.testng.annotations.Test;
 
-import static io.prestosql.spi.block.BlockTestUtils.assertBlockEquals;
 import static io.prestosql.spi.type.BigintType.BIGINT;
 
 public class TestLongArrayBlockEncoding
+        extends BaseBlockEncodingTest<Long>
```

Comment:
composition?

Reply:
What do you mean? What is wrong with inheritance here? And how would you like this composition to be implemented?